config/vufind/combined.ini

```diff
@@ -33,6 +33,14 @@
 ;         the side column. If set to false, side recommendations will be suppressed
 ;         (default). Adding the same recommendation module to multiple sections is
 ;         unsupported.
+; include_recommendations_noresults = If set to an array of recommendation settings
+;         (as per searches.ini), the specified recommendations will be loaded into
+;         the results column. If set to false, noresults recommendations will be 
```

Comment:
I think we should probably document a "true" setting here equivalent to the "true" value of include_recommendations, and make that the default behavior for better backward compatibility.

Reply:
Once #3264 is merged, I think I should be able to remove the `include_recommendations_noresults` option entirely as redundant.  But I would still introduce `include_recommendations_noresults_side`.

And in that case, I'm not sure if we need a `true` option.  I didn't include one for `include_recommendations_side`.  I see why for full compatibility they both should have it.  But I wrote in combined.ini's explanation of `include_recommendations_side` that  `Adding the same recommendation module to multiple sections is ... unsupported.` because it would lead to problems -- for the JS code moving things to the sidebar, I'm keying IDs on the recommendation module name and not the source module name, so there would be name conflicts if the same recommendation was added more than once.  And because of that I thought automatically inheriting the combined side recommendations from the parents is a bad idea, you would create conflicts without easily seeing why.  So I would rather not offer it as an option for `include_recommendations_noresults_side` either.

Reply:
#3264 is now merged, so please feel free to merge dev into here and make further adjustments -- just let me know when you're ready for the next (and hopefully final) review. Thanks!

I agree that it makes sense to omit the `true` option for `include_recommendations_noresults_side` for the reasons you state. I think there might still be some value in having an `include_recommendations_noresults` setting in case users want to disable the noresults recommendations in certain columns for some reason. This seems less likely to be changed than other types of recommendations, but if it costs us very little to make it configurable, it doesn't seem to hurt to do it while we're here anyway. :-)

Reply:
That's fair, I will leave in `include_recommendations_noresults`.  In which case I think this is ready for review again.